lib/tests/task_tests.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: avoid_print
+
 import 'package:flutter/foundation.dart';
```

Comment:
I've been trying to get tests compatible with `flutter test` in order to utilize its test suite. I've ran into an issue with mocking libraries though, since it seems like somewhere along the database class something with firebase is called even though I overrode all firebase values. I might add an entrypoint to the database class and see what happens since it is taking a while to figure out.
Ideally though, we probably want the tests to use their test suite.

Reply:
If flutter test can execute any old dart code we could definitely move tests into that. I think prof might have some more specifications about tests to bring up in class monday though so hopefully that doesnt get in the way of us using flutter test.